### PR TITLE
Final deletion date for Recycler Bin (EXPOSUREAPP-10766)

### PR DESCRIPTION
**Pull request by BMItr**
This PR adds the final deletion date to the recycler bin items.

How to test:
- delete some certificates
- go to the recycler bin inside the menu
- check if the deletion date is shown and correct

Screenshot:
image

### CODE REVIEW

**Review by KathrinLuecke — DISMISSED** [review]
UA approved

**Review by AlexanderAlferov — DISMISSED** [review]
Tested with time traveling - LGTM 🚀

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/reyclebin/ui/common/RecyclerBinCertificateItemBinding.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import de.rki.coronawarnapp.util.TimeAndDateExtensions.toShortDayFormat
+import de.rki.coronawarnapp.util.TimeAndDateExtensions.toShortTimeFormat
+import timber.log.Timber
+
+fun RecyclerBinCertificateItemBinding.addDeletionInfoIfExists(item: Recyclable) {
+    Timber.tag(TAG).v("addDeletionInfo(item=%s)", item)
+
+    certificateDeletionDateInfo.isVisible = item.recycledAt != null
+    item.recycledAt?.let { recycledAt ->
+        val deletionDate = recycledAt.plus(Recyclable.RETENTION_DAYS)
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/reyclebin/ui/common/RecyclerBinCertificateItemBinding.kt:16]: Should not we display the date/time in user time zone ? i.e `recycledAt.plus(Recyclable.RETENTION_DAYS).toLocalDateTimeUserTz()`
```kotlin
// Test was delete at 2022-01-11  00:03 (user time zone)  , 2022-01-11T23:03 UTC 
addDeletionInfo(item=PCRCoronaTest(identifier=qrcode-PCR-02EB2B-02EB2BFE-EA4D-4B94-9CE7-737063DB9EB9, registeredAt=2022-01-11T16:34:30.310Z, registrationToken=b1c6688b-d6fc-44f2-a92e-a33f40b7f9ef, isSubmitted=false, isViewed=true, didShowBadge=true, isAdvancedConsentGiven=true, isResultAvailableNotificationSent=true, testResultReceivedAt=2022-01-11T16:34:30.310Z, testResult=PCR_NEGATIVE(1), lastUpdatedAt=2022-01-11T23:03:34.423Z, isProcessing=false, lastError=null, isDccConsentGiven=true, isDccDataSetCreated=true, labId=cwa-app-cli, qrCodeHash=563bae319c58bb7f51a48adb8c2d663c482bd595568c2e2a1ac0d64b0cc4f9a0, recycledAt=2022-01-11T23:03:38.218Z))



// In user time zone
Deletion date info: wird am 11.02.22 um 00:03 endgültig gelöscht

// In UTC time zone
Deletion date info: wird am 10.02.22 um 23:03 endgültig gelösch
```

if you time travel to `10.02.22 um 23:06` , you can still see the item aren't deleted 🙅🏼. While if you time traveled to 
`11.02.22 um 00:03` items are deleted

Comment (BMItr) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/reyclebin/ui/common/RecyclerBinCertificateItemBinding.kt]: @mtwalli thats right.
adjusted.
thx

**Review by mtwalli — APPROVED** [review]
Thanks

**Review by AlexanderAlferov — APPROVED** [review]

**Review by KathrinLuecke — APPROVED** [review]
UA approved

**Pull request merged**